registry/handlers/api_test.go

```diff
@@ -57,6 +58,46 @@ func TestCheckAPI(t *testing.T) {
 	}
 }
 
+func TestURLPrefix(t *testing.T) {
```

Comment:
This might be more a test of the application rather than the API. If its easy, could this be moved to `app_test.go`?